src/animators/index.ts

```diff
@@ -0,0 +1,9 @@
+/**
```

Comment:
re-exporting is better; it allows you to reduce your API surface area. if consumers have to know about the folder structure to import symbols, then that becomes part of your public API.